website/notifications/model.py

```diff
@@ -55,8 +55,9 @@ def remove_user_from_subscription(self, user, save=True):
 
 class NotificationDigest(StoredObject):
     _id = fields.StringField(primary=True, default=lambda: str(ObjectId()))
-    user_id = fields.StringField()
+    user_id = fields.StringField(index=True)
     timestamp = fields.DateTimeField()
+    send_type = fields.StringField(index=True)
```

Comment:
Re-ping @Ghalko to address my previous comment asking about valid values and validation.